src/pages/kb/data-sources/google-sheets.md

```diff
@@ -2,8 +2,8 @@
 category: querying
 parent_category: data-sources
 helpscout_url: https://help.redash.io/article/114-querying-a-google-spreadsheet
-title: Querying a Google Spreadsheet
-slug: google-spreadsheet
+title: Google Sheets
+slug: google-sheets
```

Comment:
I think this one is missing a redirect, so that [/help/data-sources/querying/google-spreadsheet](https://deploy-preview-349--redashio.netlify.com/help/data-sources/querying/google-spreadsheet) will remain accessible.